code/modules/power/tesla/energy_ball.dm

```diff
@@ -0,0 +1,94 @@
+/obj/singularity/energy_ball
+	name = "Energy Ball"
+	icon = 'icons/obj/tesla_engine/energy_ball.dmi'
+	icon_state = "energy_ball"
+	pixel_x = -32
+	pixel_y = -32
+	current_size = STAGE_TWO
+	move_self = 1
+	grav_pull = 0
+	contained = 0
+	density = 1
+
+/obj/singularity/energy_ball/process()
+	pixel_x = 0
+	pixel_y = 0 // Lining up the beams properly.
+	tesla_zap(src, 3, 3476520)
```

Comment:
That was me and mrstonedone doing math to figure out how much power it needs to make to compete with singulo, I'll define it.